tests/xmlrpcapi/conftest.py

```diff
@@ -27,6 +27,18 @@ def token(cobbler_xmlrpc_base) -> str:
     """
     return cobbler_xmlrpc_base[1]
 
+@pytest.fixture(scope="function")
+def token2(remote: CobblerXMLRPCInterface) -> str:
+    """
+
+    :param cobbler_xmlrpc_base:
+    :return:
+    """
+    shared_secret = get_shared_secret()
+    token = remote.login("", shared_secret)
+    if not token:
+        sys.exit(1)
```

Comment:
Do we really need sys.exit instead of raising here? This would cause the whole testsuite to stop if this fixture fails, right?

Wouldn't it be safer to do something like:

```suggestion
        raise AssertionError(f"could not obtain a token when logging with {shared_secret}") 
```